config.json

```diff
@@ -72,7 +72,8 @@
     "bowling",
     "diamond",
     "all-your-base",
-    "run-length-encoding"
+    "run-length-encoding",
+		"prefect-numbers"
```

Comment:
I have no idea... 

Maybe because I added the code below at the end of the config.json file.

```
   {
      "slug": "perfect-numbers",
      "difficulty": 1,
      "topics": [
      ]
    }
```
Anyway, I corrected the typo.